R/data.R

```diff
@@ -37,7 +37,8 @@
 #'
 #' @details
 #' This metadata has its origin in the ADLB Grading Spec Excel file which ships with `{admiral}`
-#' and can be accessed using `system.file("adlb_grading/adlb_grading_spec.xlsx", package = "admiral")`.
+#' and can be accessed using `system.file("adlb_grading/adlb_grading_spec.xlsx", package = "admiral")`
+#' in sheet = "NCICTCAEv4".
```

Comment:
Can we include  the hyperlink to the NCI stuff in metadata and the function as well as hyperlink to the vignette?? 

![image](https://user-images.githubusercontent.com/10111024/200140813-e8a02daf-f930-406e-9b2d-b045f2c3a12b.png)

Reply:
sorry i see it in the function - just in the metadata then??

Reply:
Done, I think. Can you double-check I updated where you wanted?

Reply:
yes